etc/config/solidity.defaults.properties

```diff
@@ -0,0 +1,11 @@
+compilerType=solidity
+supportsBinary=false
+instructionSet=evm
+
+group.solc.isSemVer=true
+group.solc.baseName=solc
```

Comment:
Can you add a default `/usr/bin/solc` entry for the default env? That's how it's usually done for the rest of the languages

Reply:
done